### DISCUSSION TITLE
Follower Issues

### DISCUSSION THREAD

**Discussion by rpm5099** [NightscoutFoundation/xDrip#3694] [Bug reports] [answered]
Hi, I've been using xdrip for years with a variant for my daughter now for about a year.  I am recently having an issue where the follower app will either not sync or the BG values are -1.  I've tried deleting all data, reinstalling, xdrip cloud, desert sync (which I would prefer and was working prior to today) - all about 50-60 times.  I cannot get the values to reliably sync.  First picture is my daughters phone, bottom one is mine.  I think I've been through every menu and every setting, captured pcap, viewed all logs, tried several different versions and done everything I can think of over and over.  I'm getting nowhere.  If you have any idea what might be going on I'd really appreciate it.  I need to be able to view this data at all times, I'm sure you understand.  I doubt it's related, but AAPS is continually restarting (green flags, although I deleted most of them) despite the fact that all data seems to be getting received inter-app on my daughters phone.  If you need any more information please let me know.  Thanks in advance.

![Screenshot_20241003-003735](https://github.com/user-attachments/assets/65f116b1-a19e-4a40-9571-5dacf535ad2f)
![Screenshot_20241003_003231_xDrip+](https://github.com/user-attachments/assets/dee578da-4f27-4cb3-8743-9cda73677cc3)

**Comment by Navid200** [comment]
Can you post screenshots of the "Followers" status page on both phones?

**Comment by rpm5099** [comment]
@Navid200 - hey again.  Here you go - currently they are both set to use the xdrip cloud and desert sync is disabled, but I think I've tried every combination of settings imaginable.

![Screenshot_20241003_122237](https://github.com/user-attachments/assets/72b4b862-dc70-4b34-9e93-a230a097f613)
![Screenshot_20241003_121907_xDrip+](https://github.com/user-attachments/assets/a6ea0309-61fd-421e-9b12-2efceab99b87)

**Comment by jamorham** [comment]
Can you describe what is your Libre2 setup? Is the following doing its own calibrations here?

**Reply by rpm5099** [reply]
Sure, I'm using the same setup I used on my phone for several years and it worked great.  I'm running juggluco with Patched Libre broadcast to xDrip.  Xdrip is in patched libre mode, NFC is disabled on both.  Interapp settings are broadcasting locally only to "info.nightscout.aaps" and set to not receive BG, but can receive treatments.  I do have engineering mode enabled to allow the 1 minute intervals.  5 minute average for Libre is also enabled.  Only AAPS on the master sends anything to Nightscout, and that is all working correctly.

I should add that everyone on her phone is connecting and working fine, with the exception that I am getting very frequent AAPS restarts but I don't know if that's related.  Also, thats what the settings were last night and for the screenshots, I have tried *many* different combinations of settings and not had any success.

**Reply by rpm5099** [reply]
Oh sorry, forgot about calibrations - no, I have not done calibrations on either device.  I never do because it only every seems to cause problems.  Under advanced calibration everything is default except that I have it set to display plugin results on the graph, which should just be the standard xdrip (in the screenshot I did have "display plugin glucose" checked but it seems to make no difference).  I can send screenshots of settings if any of that wasn't clear.

**Reply by rpm5099** [reply, marked as answer]
Ok, somehow it started working just now after changing the calibration settings from what I stated above.  On the follower I unchecked/disabled every single setting and checkbox.  On the master I have everything disabled/unchecked *except* for "plugin plot on graph", infrequent calibration, and non-fixed libre slopes.  I assume the problem was having "use plugin glucose" checked on the master, however that is how I used it on my phone with juggluco/libre for a long time, I just never had a follower setup - perhaps a bug with calibration plugins and followers? Appreciate any insight but I'll mark this as the answer.  

Thanks again to both of you.
